packages/data/src/components/use-select/index.js

```diff
@@ -104,7 +105,100 @@ const renderQueue = createQueue();
  * ```
  * @return {UseSelectReturn<T>} A custom react hook.
  */
+function Selecter( registry ) {
+	const NOTHING = {};
```

Comment:
Yes, if it continues to be necessary (mere `undefined` doesn't work currently because we also support "really `undefined`" return value), I will extract it.

Reply:
Sounds good to me 👍